backend/main.go

```
@@ -0,0 +1,43 @@
+		if err != nil {
+			c.JSON(http.StatusInternalServerError, gin.H{"error": "Failed to fetch data from Google Books API"})
+			return
+		}
+		defer resp.Body.Close()
+
+		var result map[string]interface{}
+		json.NewDecoder(resp.Body).Decode(&result)
+		c.JSON(http.StatusOK, result)
+	})
+
+	r.Run(":8080") // サーバーを8080ポートで起動
```

golangci-lint (1.62.2)

Error return value of `r.Run` is not checked

(errcheck)